pkgs/os-specific/linux/trace-cmd/default.nix

```
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchgit, asciidoc, docbook_xsl, libxslt }:
 stdenv.mkDerivation {
   pname = "trace-cmd";
-  version = "2.9-dev";
+  version = "2.9.1+2";           # 2.9.1 + two "fix" commits
```

```suggestion
  version = "2.9.1";
```

or unstable xxxx-xx-xx format. Otherwise we introduce new versions which do not exist.

---

I can live with 2.9.1, but it's not precise. The problem is that both trace-cmd and kernelshark live in the same repository (at least for the last released versions) and their versions are not synchronized. kernelshark-1.2 is tagged few commits after trace-cmd 2.9.1.

If you think that such imprecision is not important in nixpkgs, I'll happily apply your suggestion.

---

They have release tags. We just fetch the release tag and use the version number in the tag. Then we don't need special new version numbers.

---

OK. It makes sense to use the release tags, especially given that the next version of kernel-shark will no longer share the repo with trace-cmd (see https://git.kernel.org/pub/scm/utils/trace-cmd/kernel-shark.git). I'll update the PR.